srcpkgs/Solaar/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'Solaar'
+pkgname=Solaar
+version=1.0.1
+revision=1
+build_style=python3-module
```

Comment:
You should add `pycompile_module="hidapi logitech_receiver solaar", after this line.
Also the package only contains python scripts and no architecture depended files, so setting `archs=noarch` makes sense.